lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3209,11 +3209,21 @@ namespace {
     AccessorDecl *tryCreateConstexprAccessor(const clang::VarDecl *clangVar,
                                              VarDecl *swiftVar) {
       assert(clangVar->isConstexpr());
+
+      // Taken from clang/AST/Decl.cpp
+      // We must check if this eval is not value dependent
+      // as the assert there will fail otherwise
+      clang::EvaluatedStmt *Eval = clangVar->ensureEvaluatedStmt();
+      const auto *Init = cast<clang::Expr>(Eval->Value);
+
+      // Bail if the assert in clang/AST/Decl will fail
```

Comment:
This isn't really a helpful comment. Maybe we can figure out why dependent values don't work? Or just state that dependent values don't work.